assets/scss/partials/components/_notice.scss

```diff
@@ -0,0 +1,44 @@
+.notice {
+  display:  flex;
+  flex-direction: column;
+
+  @include themed() {
+    background-color: mix(t('info'), t('accent'), 40%);
+  }
+
+  &--update {
+
```

Comment:
I assume, you don't have prettier setup correctly as this would have already removed the unnecessary whitespaces. Although we have the prettier workflow on `master` it would be cool for PRs to be formatted correctly (without running a workflow for PRs) to avoid irritation.

```suggestion
```